opal_hot_reloader.gemspec

```diff
@@ -23,14 +23,14 @@ Gem::Specification.new do |spec|
   end
 
   spec.files         = `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
-  spec.bindir        = "exe"
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir        = "bin"
+  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
   spec.add_development_dependency "bundler", "~> 1.10"
   spec.add_development_dependency "rake", "~> 10.0"
   spec.add_development_dependency "rspec"
 
-  spec.add_dependency 'listen', '3.1.1'
+  spec.add_dependency 'listen', '~> 3.0'
```

Comment:
I need to check that this works for me, I intentionally down specified because I was using a Ruby 2.2 that was < 2.2.4 - when I asked on opal/opal I see the 2 guys who replied to me also were using < 2.2.4.  I think I could ~> 3.1.1 in the gemspec and put 3.1.1 in the gemfile.  What version are you using?